common/lib/xmodule/xmodule/course_module.py

```diff
@@ -1408,3 +1417,32 @@ def clean_id(self, padding_char='='):
         return "course_{}".format(
             b32encode(unicode(self.location.course_key)).replace('=', padding_char)
         )
+
+    @property
+    def teams_enabled(self):
```

Comment:
I'm finding this method confusing as well. It seems to me like there are many layers of a course having teams "enabled".
* A course has a `teams_configuration`
* A course has a `teams_configuration` with topics

I think it'd help if you could provide an example structure of the `teams_configuration` dict.